packages/ui/src/alert.tsx

```diff
@@ -13,6 +13,7 @@ const alertVariants = cva(
         destructive:
           "border-destructive/50 text-destructive dark:border-destructive [&>svg]:text-destructive",
         warning: "border-orange-300 text-orange-300 [&>svg]:text-orange-300",
+        info: "flex flex-col gap-2 rounded-md bg-neutral-800/50 text-muted-foreground [&>svg]:top-2 [&>svg]:text-muted-foreground",
```

Comment:
```suggestion
        info: "flex flex-col gap-2 rounded-md bg-neutral-800/50 text-muted-foreground [&>svg]:top-2 [&>svg]:text-muted-foreground",
```

info should be blue - Id give it a different name